src/plugins/intel_gpu/src/kernel_selector/cl_kernels/include/batch_headers/fetch_data.cl

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
Please add a test case for b-axis padding

Reply:
Test-cases of crop for batching are added.